pt-BR/getting_started.md

```diff
@@ -187,32 +187,32 @@ of the files and folders that Rails created by default:
 |.gitignore|This file tells git which files (or patterns) it should ignore. See [GitHub - Ignoring files](https://help.github.com/articles/ignoring-files) for more info about ignoring files.
 |.ruby-version|This file contains the default Ruby version.|
 
-Hello, Rails!
+Olá, Rails!
 -------------
 
-To begin with, let's get some text up on screen quickly. To do this, you need to
-get your Rails application server running.
+Para começarmos, vamos rapidamente fazer algum texto aparecer na tela. Para fazer isso, você precisa
+colocar a seu servidor de aplicações Rails para rodar.
 
-### Starting up the Web Server
+### Iniciando o Servidor Web
 
-You actually have a functional Rails application already. To see it, you need to
-start a web server on your development machine. You can do this by running the
-following in the `blog` directory:
+Na verdade, você já tem uma aplicação Rails funcional. Para vê-la, você precisa
+iniciar um servidor web na sua máquina de desenvolvimento. Você pode fazer isso executando o
+seguinte dentro do diretório `blog`:
```

Comment:
```suggestion
seguinte comando dentro do diretório `blog`:
```

Aqui não tem a palavra "comando" no texto original, mas acredito que podemos colocar para melhorar o contexto. O que acha?

Reply:
Eu acho que faria todo sentido, já que  "`the following`" se refere a "`$ rails server`" que é um comando/código que vem a seguir do texto. 

Até mesmo no original acho que a palavra "comando" está implícita, já que "`the following`" funciona como um adjetivo e precisa de um substantivo (nesse caso "`$ rails server`") para poder fazer sentido, então mesmo no original a frase seria algo similar a "`the following command`" onde "`command`" fica implícito. Dessa forma a tradução estaria transformando um termo implícito em um temo explícito, que a meu ver deixa o texto bem mais claro e compreensível 😃